cmd/fleetctl/gitops.go

```diff
@@ -56,7 +56,14 @@ func gitopsCommand() *cli.Command {
 			logf := func(format string, a ...interface{}) {
 				_, _ = fmt.Fprintf(c.App.Writer, format, a...)
 			}
-			err = fleetClient.DoGitOps(c.Context, config, baseDir, logf, flDryRun)
+			appConfig, err := fleetClient.GetAppConfig()
+			if err != nil {
+				return err
+			}
```

Comment:
You can do the following:
https://github.com/fleetdm/fleet/blob/6e31da558b65254d8940973d0c4716966c8089f6/server/service/appconfig.go#L222-L223